server/datastore/mysql/migrations/tables/20211116184029_AddPolicyMembershipHistoryIndex.go

```diff
@@ -0,0 +1,24 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20211116184029, Down_20211116184029)
+}
+
+func Up_20211116184029(tx *sql.Tx) error {
+	_, err := tx.Exec("CREATE INDEX policy_membership_history_groupby_idx on policy_membership_history (host_id, policy_id, id)")
+	if err != nil {
+		return errors.Wrap(err, "drop app_configs table")
```

Comment:
Oops, left the copy-paste message.

```suggestion
		return errors.Wrap(err, "create policy_membership_history_groupby_idx")
```